### ISSUE TITLE
Provide a function to clear action state

### ISSUE THREAD

**Issue by Shatur** [Leafwing-Studios/leafwing_input_manager#108]
## What problem does this solve?

In my game I have a chat and pause menu, both handled by separate systems. When I press `UiAction::Back` I would like to close the chat first and on second press I would like open the menu.
So in chat system I want to catch `UiAction::Back` if it was just pressed and the chat was open, to avoid it propagation to the in-game menu.

For this use case we currently have `make_held` function which unpressed the button. I can use it to avoid action propagation. But If I switch from `just_pressed` check to `just_released` - it won't work:

```rust
if ui_actions.just_released(UiAction::Chat) {
    ui_actions.make_held(UiAction::Chat);
    println!("{}", ui_actions.just_released(UiAction::Chat)); // Prints `true`
}
```

Because the function only unpressed the button, it doesn't clear `just_*`.

## What solution would you like?

I would suggest to replace `make_held` with a new function called `reset` that fully clears the action state. So it becomes "unpressed" and "unjusted". I believe this is what user would expect and in want most cases.

**Comment by alice-i-cecile** [comment]
Agreed: I think this is much clearer.

**Comment by Shatur** [comment]
Closed by #112.
